### PR TITLE
Fixes netdata group deletion on linux for uninstall script

### PR DESCRIPTION
**Pull request by mbarper**
##### Summary
For the uninstall script, the command used to delete de netdata user in linux (userdel -f "${username}") deletes the group as well because of the -f option, so the netdata group deletion step fails.

This pull request adds a conditional to only delete the netdata group if it still exists, otherwise just print a message and mark step as OK.

##### Component Name

Uninstall script.

##### Additional Information
From userdel man page:
```
       -f, --force
           This option forces the removal of the user account, even if the user is still logged in. It also forces userdel to remove the user's home directory and mail spool, even if another user uses the
           same home directory or if the mail spool is not owned by the specified user. If USERGROUPS_ENAB is defined to yes in /etc/login.defs and if a group exists with the same name as the deleted user,
           then this group will be removed, even if it is still the primary group of another user.

           Note: This option is dangerous and may leave your system in an inconsistent state.
```

### CODE REVIEW

**Comment by paulkatsoulakis** [comment]
Hello, could you elaborate a bit more?
You are talking about userdel, but the change is on groupdel

**Comment by paulkatsoulakis** [comment]
In general, we intentionally use the `run` command that does not break the uninstall, but it rather reports on the result.  So when the group or user does not exist, it will report that it failed because it does not exist and continue

**Comment by mbarper** [comment]
As I see when I uninstalled netdata (to install it by package) a red KO message appears. I know the process finish successfully, but I had to investigate why the netdata group was deleted before that step, and I think anyone looking that message would wonder as well.
IMHO or the groupdel section dissapear or we control if the group is present or not to show the message (this is the change I propose).

**Comment by mbarper** [comment]
Don't you think it is a good practice? Just to know if this PR is goingo to progress or forget it.

File: packaging/installer/netdata-uninstaller.sh [hunk 1]
```diff
@@ -158,7 +158,12 @@ portable_del_group() {
 
 	# Linux
 	if command -v groupdel 1>/dev/null 2>&1; then
-		run groupdel -f "${groupname}" && return 0
+		if grep -q "${groupname}" /etc/group; then
+		  run groupdel -f "${groupname}" && return 0
+		else
+		  echo >&2 "Group ${groupname} already remove in a previous step."
```
Comment (cakrit) [inline packaging/installer/netdata-uninstaller.sh]: plz change "remove" to "removed"

**Comment by cakrit** [comment]
It's fine, just plz correct the message grammar and we will merge.

**Comment by mbarper** [comment]
Typo solved. Thanks!

File: packaging/installer/netdata-uninstaller.sh (hunk 1 above)
Comment (mbarper) [reply packaging/installer/netdata-uninstaller.sh]: Done. Thanks!

**Comment by cakrit** [comment]
No, thank YOU @mbarper!

**Pull request merged**